app/Http/Controllers/Device/Tabs/ShowConfigController.php

```diff
@@ -70,9 +70,29 @@ public function data(Device $device): array
 
     private function oxidizedEnabled(Device $device)
     {
-        return Config::get('oxidized.enabled') === true
-            && Config::has('oxidized.url')
-            && $device->getAttrib('override_Oxidized_disable') !== 'true';
+        $enabled = false;
+
+        $enabled = Config::get('oxidized.enabled') === true
+                && Config::has('oxidized.url')
+                && $device->getAttrib('override_Oxidized_disable') !== 'true';
+
+        if (Config::has('oxidized.ignore_types.0')) {
+            foreach (Config::get('oxidized.ignore_types') as $ignoreType) {
+                if ($ignoreType == $device->type) {
+                    $enabled = false;
+                }
+            }
+        }
+
+        if (Config::has('oxidized.ignore_os.0')) {
+            foreach (Config::get('oxidized.ignore_os') as $ignoreOS) {
+                if ($ignoreOS == $device->os) {
+                    $enabled = false;
+                }
+            }
+        }
+
+        return $enabled;
```

Comment:
this could be simplified as something like 
```suggestion
        return Config::get('oxidized.enabled') === true
                && Config::has('oxidized.url')
                && $device->getAttrib('override_Oxidized_disable') !== 'true'
                && ! in_array($device->type, Config::get('oxidized.ignore_types', []))
                && ! in_array($device->os, Config::get('oxidized.ignore_os', []));
```